docs/develop/api-reference/index.mdx

```diff
@@ -445,3 +451,13 @@ raise response.error if response.error?
   `}
   cli={`momento cache delete --key test-key --value test-value`}
 />
+
+### Collection data types (CDTs)
```

Comment:
do you use `CDT` elsewhere in the docs?  If not maybe leave that out for now?

Reply:
Yes. It is used on the main data types page.